src/utils/monitoring-sites-data.js

```diff
@@ -104,4 +119,5 @@ export const displayMonitoringSitesFeaturesInMapViewport = async (map) => {
   await setSitesInBoundingBox(map, corners, "waterQualitySites");
   await setSitesInBoundingBox(map, corners, "riverLevelSites");
   await setSitesInBoundingBox(map, corners, "freshwaterSites");
+  await setRiverFlowSitesInBoundingBox(map, corners, "riverFlowSites");
```

Comment:
nitpick but we can probably avoid passing the third `"riverFlowSites"` argument here since there's already a bespoke function for those ones in particular.

Reply:
ooh yeah, makes sense - thank you!